lldb/test/API/python_api/global_module_cache/TestGlobalModuleCache.py

```diff
@@ -111,6 +111,8 @@ def do_test(self, one_target, one_debugger):
         else:
             if one_target:
                 new_debugger = lldb.SBDebugger().Create()
+                if lldb.selected_platform is not None:
+                    new_debugger.SetSelectedPlatform(lldb.selected_platform)
```

Comment:
```suggestion
                new_debugger.SetSelectedPlatform(lldb.selected_platform)
```

The selected platform is set unconditionally, no need to hedge here.

Reply:
Thanks! I copied the code from `Base.setUp()` (`lldbtest.py`) because I thought the condition was there for a reason.